pycsw/ogc/api/templates/item.html

```diff
@@ -41,76 +46,77 @@ <h2>{{ data['id'] }}</h2>
         <tbody>
           {% if 'properties' in data %}
             {% for key, value in data['properties'].items() %}
-            <tr>
-              <td>{{ key | capitalize }}</td>
-              {% if key == 'keywords' or key == 'formats' %}
-                <td>
-                  <ul>
-                  {% for keyword in value %}
-                    <li>{{ keyword }}</li>
-                  {% endfor %}
-                  </ul>
-                </td>
-
-              {% elif key == 'themes' %}
-                <td>
-                {% for theme in value %}
-                  <b>{{ theme['scheme'] }}</b>
-                  <ul>
-                  {% for concept in theme['concepts'] %}
-                    <li>
-                      {% if concept['url'] %}
-                        <a href="{{ concept['url'] }}">{{ concept['name'] or concept['url'] }}</a>
-                      {% else %}
-                        {{ concept['name'] }}
-                      {% endif %}
-                    </li>
-                  {% endfor %}
-                  </ul>
-                {% endfor %}
-                </td>
-              {% elif key == 'externalIds' %}
-                <td>
-                  <ul>
-                    {% for id in value %}
-                    <li>{{ id['scheme'] | urlize }} {{ id['value'] | urlize }}</li>
+              {% if key not in ['title', 'description', 'type'] %}
```

Comment:
I think we should still keep those fields included in the properties, even if they are rendered above

Reply:
isn't that a duplication? mostly for description, that field can have a lot of text

(ok for type and title, maybe first 100chars of description?)

Reply:
The user needs to realize that those fields are included in the JSON properties. Yes, this leads to duplication. Thanks for the update!